### ISSUE TITLE
Does not Run

### ISSUE THREAD

**Issue by thedillonb** [moul/docker-readthedocs#6]
It looks like this container does not work correctly any more. Attempting to do a `make run` results in the following:

```
+ PYTHON=/venv/bin/python
+ /venv/bin/python manage.py syncdb --noinput
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
+ /venv/bin/python manage.py migrate
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
+ /venv/bin/python manage.py shell
+ echo 'from django.contrib.auth.models import User; User.objects.create_superuser('\''admin'\'', '\''admin@localhost'\'', '\''admin'\'')'
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
+ /venv/bin/python manage.py loaddata test_data
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
+ /venv/bin/python manage.py makemessages --all
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
+ /venv/bin/python manage.py compilemessages
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
+ /venv/bin/python manage.py celeryd -l INFO
+ /venv/bin/python manage.py runserver 0.0.0.0:8000
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
/venv/bin/python: can't open file 'manage.py': [Errno 2] No such file or directory
make: *** [run] Error 2
```

Then abruptly stops.

**Comment by moul** [comment]
Hi @thedillonb, I need to investigate, I think readthedocs changed its files layout again

**Comment by moul** [comment]
Fixed with #7 

Can you try again ?
